internal/pkg/levee/propagation/propagation.go

```diff
@@ -43,22 +43,22 @@ type Propagation struct {
 
 // Dfs performs a depth-first search of the graph formed by SSA Referrers and
```

Comment:
The name in the comment no longer matches the function's name.

Reply:
Thank for pointing this out, Ill fix this once we decide on the names of the functions of this package.

Reply:
Done.